apps/google-analytics-4/frontend/test/mocks/mockCma.ts

```diff
@@ -1,6 +1,11 @@
 const mockCma: any = {
   appSignedRequest: {
-    create: () => ({}),
+    create: () =>
+      Promise.resolve({
+        additionalHeaders: 'lol',
+        'X-Contentful-ServiceAccountKeyId':
+          'eyJpZCI6IlBSSVZBVEVfS0VZX0lEIiwiY2xpZW50SWQiOiJDTElFTlRfSUQiLCJjbGllbnRFbWFpbCI6IkNMSUVOVF9JRCIsInByb2plY3RJZCI6IlBST0pFQ1RfSUQifQ==',
```

Comment:
thinking about this more, I wonder if we should just use a string of the same length? not because the header representation of this service test key is so sensitive, but just so that anyone looking at this in the future doesn't wonder if we added an actually in-use key?

Reply:
I think we could just remove this change entirely now that I went about resolving the test failures differently.